tests/e2e/resources/drivers/drivers.txt

```diff
@@ -0,0 +1,2 @@
+geckodriver=0.35.0
```

Comment:
It's high time, we have a repository in our org caching the dependencies we use. It'll prevent bloating of PRs and unnecessary code from being checked in.
I'll get on this and keep you posted.

Reply:
I've added the webdrivers to this repo which we can use as our dependency cache henceforth.
https://github.com/padaiyal/depot/tree/main/webdriver